src/v2/Apps/Fair/Components/FairHeader.tsx

```diff
@@ -96,6 +112,8 @@ export const FairHeaderFragmentContainer = createFragmentContainer(FairHeader, {
       ticketsLink
       hours
       links
+      tickets
```

Comment:
Should this be `tickets(format: HTML)` if it's being used with `<HTML>` below?

Reply:
Ah, I was thinking it didn't matter (we don't actually use the value here), but you're right that if we want it to be cached it should be the same.

Reply:
Ah, I see; surprised this isn't erroring in that case — different page I guess?

Reply:
Yeah-- the only reason these fields are here is to decide whether to show the `More info` button or not, so the formatting doesn't matter.